lib/customer.rb

```diff
@@ -0,0 +1,35 @@
+require 'csv'
+require 'awesome_print'
+
+class Customer
+    attr_reader :id
+    attr_accessor :email, :address
```

Comment:
`email` and `address don't need to be accessors.  Default to creating readers over accessors unless the user needs to be able to directly change the instance variables.